src/containers/Category.js

```diff
@@ -209,30 +250,53 @@ function postsProp(state, listId, page) {
  */
 function mapStateToProps(state, props) {
   const location = _.get(props, 'location')
-  const pathSegment = _.get(props, 'match.params.category')
-  const catId = categoryConst.ids[pathSegment]
-  const catLabel = categoryConst.labels[pathSegment]
-  const pathname = _.get(location, 'pathname', `/categories/${pathSegment}`)
+  const categoryPath = _.get(props, 'match.params.category')
+  const subcategoryPath = _.get(props, 'match.params.subcategory')
+  const category = CATEGORY[categoryPath]
+  const subcategory = SUBCATEGORY[subcategoryPath]
 
+  let listId = `${category && category.id}`
```

Comment:
how about simply `category?.id`

Reply:
bable 6 do not deal with optional chaining, I'll create another PR to upgrade babel then fix this 😂